src/v3/hooks/useGlobal.tsx

```diff
@@ -68,7 +68,13 @@ const useGlobal = () => {
         apiHost = import.meta.env.VITE_BACKEND_API;
       } else {
         if (import.meta.env.DEV || window.location.host.indexOf('localhost') !== -1) {
-          apiHost = 'http://localhost:8000/';
+          if (import.meta.env.API_HOST_PORT) {
```

Comment:
@Logan08691: I think the `VITE_BACKEND_API` env should already be enough for overriding this api check. Thank you.